xml/en/docs/community.xml

```diff
@@ -0,0 +1,136 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="Welcome to the NGINX Community!"
```

Comment:
The information in this new page heavily overlaps with the content and intent of the 'support' page. There seems to be no good reason to introduce a new page here. I recommend that 'support' is renamed to 'community' (and on the menu) and the relevant sections of this file is carefully merged with the existing content.

A short-term redirect may be needed for /en/support.html

Reply:
Mentioned explicitly in commit message as requested